pagopa/pn-frontend
# New payment api - pn 2804

## Short description
When user click on payment button, a new api must be called. This api returns the url to which navigate

## List of changes proposed in this pull request
- Added new route in notifications.route.ts
- Added new api call in Notifications.api.ts
- Added new action in actions.ts
- Added new dispatch call in NotificationPayment.tsx

## How to test
Run pf webapp pointing to dev environment -> log with ada user -> search notification with iun PGYD-JDQY-RPZT-202208-L-1 -> click on payment button and check that user will be redirect to checkout page -> try error and success payment and check that user will be redirect to the notification detail page
